hrqb/tasks/pipelines.py

```diff
@@ -1,3 +1,17 @@
 """hrqb.tasks.pipelines"""
 
 # WIP: HRQBPipelineTasks will be defined here
+from collections.abc import Iterator
+
+import luigi  # type: ignore[import-untyped]
+
+from hrqb.base.task import HRQBPipelineTask
+
+
+class FullUpdate(HRQBPipelineTask):
+    """Pipeline to perform a full update of all Quickbase tables."""
+
+    def requires(self) -> Iterator[luigi.Task]:  # pragma: no cover
+        from hrqb.tasks.employees import LoadEmployees
```

Comment:
Imports are performed inside the `requires()` method to avoid them imported at the top level.  This keeps imports from _this_ file limited to `HRQBPipelineTask` tasks themselves.